packages/preview/lemmify-0.1.1/typst.toml

```diff
@@ -0,0 +1,8 @@
+[package]
+name = "lemmify"
+version = "0.1.1"
+entrypoint = "src/lib.typ"
+authors = ["Marmare314"]
+license = "GPL-3.0-only"
+description = "A library for typesetting mathematical theorems."
```

Comment:
It's not yet a guideline, but the "A library" doesn't really anything and makes the package list less useful, especially in mobile. Maybe this could be reworded?

![Package list](https://cdn.discordapp.com/attachments/1088371867913572452/1126244428605030644/image.png)